app-office/planner/planner-9999.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="Project manager for Gnome"
+HOMEPAGE="https://wiki.gnome.org/Apps/Planner"
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_REPO_URI="https://gitlab.gnome.org/World/planner.git"
+	inherit git-r3
+	SRC_URI=""
+else
+	KEYWORDS="~amd64"
+fi
+
+SLOT="0"
+LICENSE="GPL-2+"
+IUSE="examples gtk-doc libgda" # eds
+
+RDEPEND="
+	>=dev-libs/glib-2.50:2
+	>=x11-libs/gtk+-3.10:3
+	>=dev-libs/libxml2-2.6.27:2
+	>=dev-libs/libxslt-1.1.23
+	database-gda? ( >=gnome-extra/libgda-1.0:5 )
+"
+	# eds support is broken for now
+	# eds? (
+	# 	>=gnome-extra/evolution-data-server-3.6:=
+	# 	>=mail-client/evolution-3.6
+	# )
+DEPEND="${RDEPEND}"
+
+BDEPEND="use gtk-doc? ( dev-util/gtk-doc )"
```

Comment:
`use`? This too probably barfs portage, recheck things actually work when reinstalling

Reply:
oh shit. gosh I am tired :(

Reply:
Also with gtk-doc you almost also always need at least one app-text/docbook-xml-dtd slot, depending on what upstream uses in its gtk-doc xml files. In case of planner, this is app-text/docbook-xml-dtd:4.1.2 right now, determined by this from `libplanner-docs.xml`:

```xml
<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN"
               "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd">
```